src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java

```diff
@@ -80,4 +85,14 @@
         return new ResponseEntity <>(new MessageDTO(noSuchElementException.getMessage()), HttpStatus.NOT_FOUND);
     }
 
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex, HttpHeaders headers, HttpStatus status, WebRequest request) {
+        List<FieldErrorDTO> errors = new ArrayList<>();
+        ValidationMessageDTO messageDTO = new ValidationMessageDTO(errors);
+        ex.getBindingResult().getFieldErrors().stream().forEach((error) -> {
+            errors.add(new FieldErrorDTO(error.getField(), error.getDefaultMessage()));
```

Comment:
If you're using stream, use it the right way. It should be stream, map, collect(to list).